mps_youtube/helptext.py

```diff
@@ -79,6 +79,8 @@ def helptext():
     {2}dvupl <username>{1} - download user's YouTube playlists (video).
     {2}dlurl <url or id>{1} download a YouTube video by url or video id.
     {2}playurl <url or id>{1} play a YouTube video by url or id.
+    {2}browserplay <url or id>{1} play a YouTube video by url or id in browser.
```

Comment:
Wouldn't it make sense for this to accept numbers as well? Like `browserplay 1` to open the first result in the browser.